aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -1183,6 +1173,15 @@ module aptos_framework::stake {
         // Pending active stake can now be active.
         coin::merge(&mut stake_pool.active, coin::extract_all(&mut stake_pool.pending_active));
 
+        // Additionally, distribute transaction fees.
```

Comment:
cc @movekevin this part has to be here to make sure rewards are processed the same way as before